### PR TITLE
Grammatical fixes and improvisations

### PR DESCRIPTION
**Pull request by VroomRk**
Adhered to the guidelines and fixed minor grammatical mistakes and improved a few sentences and made them concise.

I hope you don't mind me creating a pull request directly before creating an issue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -279,19 +279,19 @@ To install using pip, enter the following command at a Bash or Windows command p
...
+Check the [installation documentation](https://docs.bokeh.org/en/latest/docs/first_steps/installation.html) for further details.
 
 ## Resources
 
-Once Bokeh is installed, check out the [first steps guides](https://docs.bokeh.org/en/latest/docs/first_steps.html#first-steps-guides).
+Once Bokeh is installed, check out the [first step guides](https://docs.bokeh.org/en/latest/docs/first_steps.html#first-steps-guides).
 
-Visit the [full documentation site](https://docs.bokeh.org) to view the [User's Guide](https://docs.bokeh.org/en/dev/docs/user_guide.html) or [launch the Bokeh tutorial](https://mybinder.org/v2/gh/bokeh/bokeh-notebooks/master?filepath=tutorial%2F00%20-%20Introduction%20and%20Setup.ipynb) to learn about Bokeh in live Jupyter Notebooks.
+To view the [User's Guide](https://docs.bokeh.org/en/dev/docs/user_guide.html), go to the [full documentation site](https://docs.bokeh.org), or [launch the Bokeh tutorial](https://mybinder.org/v2/gh/bokeh/bokeh-notebooks/master?filepath=tutorial%2F00%20-%20Introduction%20and%20Setup.ipynb) to learn about Bokeh in live Jupyter Notebooks.
```
Comment (bryevdv) [inline README.md:288]: The order of this sentence was intentional, have parallel narrative structure: " Do A for B, or do C for D". Please revert this change.

File: README.md [hunk 2]
```diff
@@ -268,7 +268,7 @@
 
 ## Installation
 
-The easiest way to install Bokeh is using the [Anaconda Python distribution](https://www.anaconda.com/what-is-anaconda/) and its included *Conda* package management system. To install Bokeh and its required dependencies, enter the following command at a Bash or Windows command prompt:
+The [Anaconda Python distribution](https://www.anaconda.com/what-is-anaconda/) and its included *Conda* package management system are the simplest way to install Bokeh. Enter the following command at a Bash or Windows command prompt to install Bokeh and its dependencies:
```
Comment (bryevdv) [inline README.md:271]: The order of this sentence was intentional, to lead with and emphasize "easy way to install Bokeh". Pleas revert this change.

**Review by bryevdv — COMMENTED** [review]
Hi @VroomRk thanks for your PR. I do have a number of concerns with some of the edits. Comments are left inline. @tcmetzger may also want to chime in

File: README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 ----
 
-[Bokeh](https://bokeh.org) is an interactive visualization library for modern web browsers. It provides elegant, concise construction of versatile graphics, and affords high-performance interactivity over large or streaming datasets. Bokeh can help anyone who would like to quickly and easily make interactive plots, dashboards, and data applications.
+[Bokeh](https://bokeh.org) is an interactive visualization library for modern web browsers. It provides concise construction of versatile graphics and high-performance interactivity across large or streaming datasets.  Anyone who wants to create interactive charts, dashboards, and data apps quickly and simply may do so with the aid of Bokeh.
```
[the remark this answers is not in the document]
Comment (VroomRk) [reply README.md]: - Added elegant back
- restored affords
- Changed the second sentence's syntax to the previous version but I guess there is a lack of fluency, and I made a minor improvisation.

File: README.md [hunk 4]
```diff
@@ -124,7 +124,7 @@
 
 </table>
 
-*If you like Bokeh and would like to support our mission, please consider [making a donation](https://numfocus.org/donate-to-bokeh).*
+*If you enjoy Bokeh and want to help us further our mission, please consider [making a donation](https://numfocus.org/donate-to-bokeh).*
```
[the remark this answers is not in the document]
Comment (VroomRk) [reply README.md:127]: - Enjoy using Bokeh is perfect. Like is still fine but "enjoy using" sounds perfect
- Another Idea I have is, to start the sentence with the main subject. "Consider making a donation if you enjoy using Bokeh and want to support its development."

File: README.md [hunk 5]
```diff
@@ -279,19 +279,19 @@ To install using pip, enter the following command at a Bash or Windows command p
...
 ## Resources
 
-Once Bokeh is installed, check out the [first steps guides](https://docs.bokeh.org/en/latest/docs/first_steps.html#first-steps-guides).
+Once Bokeh is installed, check out the [first step guides](https://docs.bokeh.org/en/latest/docs/first_steps.html#first-steps-guides).
 
-Visit the [full documentation site](https://docs.bokeh.org) to view the [User's Guide](https://docs.bokeh.org/en/dev/docs/user_guide.html) or [launch the Bokeh tutorial](https://mybinder.org/v2/gh/bokeh/bokeh-notebooks/master?filepath=tutorial%2F00%20-%20Introduction%20and%20Setup.ipynb) to learn about Bokeh in live Jupyter Notebooks.
+To view the [User's Guide](https://docs.bokeh.org/en/dev/docs/user_guide.html), go to the [full documentation site](https://docs.bokeh.org), or [launch the Bokeh tutorial](https://mybinder.org/v2/gh/bokeh/bokeh-notebooks/master?filepath=tutorial%2F00%20-%20Introduction%20and%20Setup.ipynb) to learn about Bokeh in live Jupyter Notebooks.
 
-Community support is available on the [Project Discourse](https://discourse.bokeh.org).
+On the [Project Discourse](https://discourse.bokeh.org), you can find community support.
```
[the remark this answers is not in the document]
Comment (VroomRk) [reply README.md:290]: Makes sense, reverting this change.

File: README.md [hunk 6]
```diff
@@ -279,19 +279,19 @@ To install using pip, enter the following command at a Bash or Windows command p
 pip install bokeh
 ```
 
-For more information, refer to the [installation documentation](https://docs.bokeh.org/en/latest/docs/first_steps/installation.html).
+Check the [installation documentation](https://docs.bokeh.org/en/latest/docs/first_steps/installation.html) for further details.
 
 ## Resources
 
-Once Bokeh is installed, check out the [first steps guides](https://docs.bokeh.org/en/latest/docs/first_steps.html#first-steps-guides).
+Once Bokeh is installed, check out the [first step guides](https://docs.bokeh.org/en/latest/docs/first_steps.html#first-steps-guides).
```
[the remark this answers is not in the document]
Comment (VroomRk) [reply README.md:286]: My major concern here is, that you can't include two plural words together, that sounds off,  I would like to suggest changing "first steps guides" to "first steps guide".

File: README.md [hunk 7]
```diff
@@ -279,19 +279,19 @@ To install using pip, enter the following command at a Bash or Windows command p
 pip install bokeh
 ```
 
-For more information, refer to the [installation documentation](https://docs.bokeh.org/en/latest/docs/first_steps/installation.html).
+Check the [installation documentation](https://docs.bokeh.org/en/latest/docs/first_steps/installation.html) for further details.
```
[the remark this answers is not in the document]
Comment (VroomRk) [reply README.md:282]: Agreed 👍

File: README.md [hunk 8]
```diff
@@ -279,19 +279,19 @@ To install using pip, enter the following command at a Bash or Windows command p
...
+To view the [User's Guide](https://docs.bokeh.org/en/dev/docs/user_guide.html), go to the [full documentation site](https://docs.bokeh.org), or [launch the Bokeh tutorial](https://mybinder.org/v2/gh/bokeh/bokeh-notebooks/master?filepath=tutorial%2F00%20-%20Introduction%20and%20Setup.ipynb) to learn about Bokeh in live Jupyter Notebooks.
 
-Community support is available on the [Project Discourse](https://discourse.bokeh.org).
+On the [Project Discourse](https://discourse.bokeh.org), you can find community support.
 
 If you would like to contribute to Bokeh, please review the [Contributor Guide](https://docs.bokeh.org/en/latest/docs/dev_guide.html) and [request an invitation to the Bokeh Dev Slack workspace](https://slack-invite.bokeh.org/).
 
-*Note: Everyone interacting in the Bokeh project's codebases, issue trackers and discussion forums is expected to follow the [Code of Conduct](https://github.com/bokeh/bokeh/blob/master/CODE_OF_CONDUCT.md).*
+*Note: Everyone who interacts with the Bokeh project's codebases, issue trackers, and discussion forums is expected to follow the [Code of Conduct](https://github.com/bokeh/bokeh/blob/master/CODE_OF_CONDUCT.md).*
```
[the remark this answers is not in the document]
Comment (VroomRk) [reply README.md:294]: Thanks for the context, I feel the original phrasing doesn't communicate the right message,  I have an idea to improve this would love to have your thoughts after I commit.

File: README.md [hunk 9]
```diff
@@ -368,7 +368,7 @@ If your company uses Bokeh and is able to sponsor the project, please contact <a
 
 ### In-kind Support
 
-The Bokeh project is grateful for the donation of services and time from the following companies, supporting development as well as our collaboration, infrastructure, security, and vulnerability management:
+The Bokeh project is grateful to the following companies for donating services and time to help with development as well as collaboration, infrastructure, security, and vulnerability management:
```
[the remark this answers is not in the document]
Comment (VroomRk) [reply README.md:371]: You are right, "non-monetary" is the best word to compare, and splitting the sentence is sensible.

File: README.md (hunk 1 above)
Comment (VroomRk) [reply README.md:288]: reverting this

File: README.md (hunk 2 above)
Comment (VroomRk) [reply README.md:271]: Reverting this as well

File: README.md [hunk 10]
```diff
@@ -124,7 +124,7 @@
 
 </table>
 
-*If you like Bokeh and would like to support our mission, please consider [making a donation](https://numfocus.org/donate-to-bokeh).*
+*Consider [making a donation](https://numfocus.org/donate-to-bokeh). if you enjoy using Bokeh and want to support its development.*
```
Comment (bryevdv) [inline README.md:127]:
```suggestion
*Consider [making a donation](https://numfocus.org/donate-to-bokeh) if you enjoy using Bokeh and want to support its development.*
```

File: README.md [hunk 11]
```diff
@@ -368,7 +368,9 @@ If your company uses Bokeh and is able to sponsor the project, please contact <a
 
 ### In-kind Support
 
-The Bokeh project is grateful for the donation of services and time from the following companies, supporting development as well as our collaboration, infrastructure, security, and vulnerability management:
+Non-monetary support can help with development, collaboration, infrastructure, security, and vulnerability management.
+
+The Bokeh project is grateful to the following companies for their donation of services:
```
Comment (bryevdv) [inline README.md:373]:
```suggestion
Non-monetary support can help with development, collaboration, infrastructure, security, and vulnerability management. The Bokeh project is grateful to the following companies for their donation of services:
```
 
I don't think a new para is needed here

File: README.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (bryevdv) [reply README.md]: @tcmetzger do you have any strong feelings here?

[the remark this answers is not in the document]
Comment (bryevdv) [reply README.md]: Alternatively, a hyphen could help here "first-steps" to indicate that it is a collective noun

**Comment by bryevdv** [comment]
@VroomRk Changes look good, only a few cosmetic comments now, plus I'd like @tcmetzger to express his preference re: "first steps"

File: README.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (VroomRk) [reply README.md]: A hyphen is sensible and it would reflect the intended meaning, but "First-steps" should be preceded by "guide" instead of "guides", do you think this makes sense?

File: README.md (hunk 10 above)
Comment (VroomRk) [reply README.md:127]: Accepting your suggestion

File: README.md (hunk 11 above)
Comment (VroomRk) [reply README.md:373]: Yes, Accepting this suggestion as well.

**Comment by VroomRk** [comment]
Thanks, @bryevdv ,  I would also like to have more opinions on  "first-steps guides"

File: README.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (bryevdv) [reply README.md]: @VroomRk I have thought a little more about this and I think "first steps guides" is the best expression of the correct meaning.  For one there are *multiple* guides (nine of them in fact):
Screen Shot 2022-08-11 at 11 24 58

that all provide information about various and different first steps with Bokeh. I am not sure what prescriptivist rule  bars "plurals in a row", but:

* "first steps guides" is completely comprehensible and in fact does not even sound strange or unusual to my ear 
* English is malleable and forgiving, and sometimes the application of judgment is more important to convey the most accurate meaning

Also looking at the other page, I realize adding a hyphen only here would be inconsistent, so I think we should forego that as well. Please revert to the original "first steps guides" and then I think this will be ready to merge.

File: README.md [hunk 12]
```diff
@@ -4,7 +4,7 @@
 
 ----
 
-[Bokeh](https://bokeh.org) is an interactive visualization library for modern web browsers. It provides elegant, concise construction of versatile graphics, and affords high-performance interactivity over large or streaming datasets. Bokeh can help anyone who would like to quickly and easily make interactive plots, dashboards, and data applications.
+[Bokeh](https://bokeh.org) is an interactive visualization library for modern web browsers. It provides elegant, concise construction of versatile graphics and affords high-performance interactivity across large or streaming datasets. Bokeh can help anyone who wants to create interactive plots, dashboards, and data applications quickly and easily.
```
Comment (bryevdv) [inline README.md:7]: BTW @VroomRk we have tried to keep this "elevator pitch" consistent across all the bokeh.org properties.  After this PR is merged, if you could submit a PR to update the "front page" to match, that would be appreciated:

https://github.com/bokeh/bokeh.org/blob/2f082bb9fa8691547c593d5e6790b74356b7aa97/_includes/roadmap.html#L12-L14

That would be appreciated. There may also be a similar statement to update in the Sphinx source for the docs page too.

File: README.md [hunk 13]
```diff
@@ -279,19 +279,19 @@ To install using pip, enter the following command at a Bash or Windows command p
...
+Once Bokeh is installed, check out the [first steps guide](https://docs.bokeh.org/en/latest/docs/first_steps.html#first-steps-guides).
 
 Visit the [full documentation site](https://docs.bokeh.org) to view the [User's Guide](https://docs.bokeh.org/en/dev/docs/user_guide.html) or [launch the Bokeh tutorial](https://mybinder.org/v2/gh/bokeh/bokeh-notebooks/master?filepath=tutorial%2F00%20-%20Introduction%20and%20Setup.ipynb) to learn about Bokeh in live Jupyter Notebooks.
 
 Community support is available on the [Project Discourse](https://discourse.bokeh.org).
 
 If you would like to contribute to Bokeh, please review the [Contributor Guide](https://docs.bokeh.org/en/latest/docs/dev_guide.html) and [request an invitation to the Bokeh Dev Slack workspace](https://slack-invite.bokeh.org/).
 
-*Note: Everyone interacting in the Bokeh project's codebases, issue trackers and discussion forums is expected to follow the [Code of Conduct](https://github.com/bokeh/bokeh/blob/master/CODE_OF_CONDUCT.md).*
+*Everyone who engages in the Bokeh project's discussion forums, codebases, and issue trackers is expected to follow the [Code of Conduct].*
```
Comment (bryevdv) [inline README.md:294]: I think the link got accidentally dropped here?

Comment (bryevdv) [reply README.md:294]: That said, the link did need updating to "main":

https://github.com/bokeh/bokeh/blob/main/CODE_OF_CONDUCT.md

is the correct link to use now

File: README.md (hunk 12 above)
Comment (VroomRk) [reply README.md:7]: Sure @bryevdv I will submit another PR to make these changes 👍

File: README.md (hunk 13 above)
Comment (VroomRk) [reply README.md:294]: Yes, this is an accident I have updated the link.

File: README.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (VroomRk) [reply README.md]: Sure Bryan. I can understand that "first steps guides" don't change the intended meaning. I will revert the changes.

**Comment by mattpap** [comment]
> Grammatical fixes and improvisations

I suppose you've meant "Grammatical fixes and improvements"?

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
thanks @VroomRk !
